packages/flutter_tools/lib/src/commands/run.dart

```diff
@@ -187,9 +187,12 @@ class RunCommand extends RunCommandBase {
         hide: true,
         help: 'Whether to automatically invoke webOnlyInitializePlatform.',
       )
+      // TODO(jonahwilliams): Off by default with investigating whether this
+      // is slower for certain use cases.
+      // See: https://github.com/flutter/flutter/issues/49499
       ..addFlag('fast-start',
         negatable: true,
-        defaultsTo: true,
+        defaultsTo: false,
```

Comment:
Should we also hide it again?

Reply:
I would like to keep it visible so that I can more easily determine if it is useful to developers on slower machines